easybuild/easyconfigs/c/canu/canu-1.8-foss-2018b.eb

```diff
@@ -0,0 +1,36 @@
+easyblock = 'MakeCp'
+
+name = 'canu'
+version = '1.8'
+
+homepage = 'http://canu.readthedocs.io'
+description = """Canu is a fork of the Celera Assembler designed for
+ high-noise single-molecule sequencing"""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+
+source_urls = ['https://github.com/marbl/canu/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['30ecfe574166f54f79606038830f68927cf0efab33bdc3c6e43fd1448fa0b2e4']
+
+dependencies = [('Boost', '1.67.0')]
```

Comment:
@fizwit I've had a PR open for a while for this, see #7333

I used different dependencies, not including `Boost`. Any reason you didn't include `Perl` or `gnuplot`?

Reply:
This is an update of something I built a few years ago.  I have never used the software so can not comment. It builds without error, but have no idea how it functions. If you push your 2018b version I will use it.

Reply:
@fizwit I tested the installation of `canu` without having Boost available (either as a dependency, or on the system), so as far as I can tell it doesn't require `Boost`.

@verdurin Shall we go ahead and merge #7333 based on this feedback, so @fizwit can update this PR to be in sync with #7333?

Reply:
@fizwit Ping on updating this PR to be in sync with #7333?